doc/ref_model/chapters/chapter03.md

```diff
@@ -543,33 +543,52 @@ SyncE architecture minimally requires replacement of the internal clock of the E
 ## 3.6 Storage
 The general function of storage subsystem is to provide the persistent data store required for the delivery of a network service. In the context of NFVi the storage sub-system needs to accommodate needs of: the tenanted VNF applications and the platform management.
```

Comment:
@jbhartley To keep the document uniformity, use "Cloud Infrastructure" rather than "NFVi"

Reply:
Corrected.